vendor/composer/autoload_static.php

```diff
@@ -26,13 +26,6 @@ class ComposerStaticInit453516e196e1be4c43fa495b9bb9d572
     public static $classMap = array (
         'WP_Async_Request' => __DIR__ . '/..' . '/a5hleyrich/wp-background-processing/classes/wp-async-request.php',
         'WP_Background_Process' => __DIR__ . '/..' . '/a5hleyrich/wp-background-processing/classes/wp-background-process.php',
-        'tad_DI52_Container' => __DIR__ . '/..' . '/lucatume/di52/src/tad/DI52/Container.php',
```

Comment:
This should not be removed...

Reply:
hm...

Reply:
No matter how I try to work composer, it keeps getting removed :(